akd/Cargo.toml

```diff
@@ -32,18 +34,19 @@ bincode = { version = "1", optional = true }
 serde = { version = "1", features = ["derive"], optional = true }
 rand = { version = "0.7", optional = true }
 curve25519-dalek = { version = "3.2", optional = true }
-ed25519-dalek = { version = "1", optional = true}
-
-[target.'cfg(all(features = "serde", features = "vrf"))'.dependencies]
```

Comment:
This is apparently not supported (and had typos anyways). Need to have builder utilize features on the crate to support it.